README.md

```diff
@@ -37,42 +53,32 @@
 
 ## Introduction
 
-Teleport is the easiest, most secure way to access all your infrastructure.
-Teleport is an identity-aware, multi-protocol access proxy which understands
-SSH, HTTPS, RDP, Kubernetes API, MySQL, MongoDB and PostgreSQL wire protocols.
+Teleport includes an identity-aware access proxy, a CA that issues short-lived certificates, and an access control and tunneling system to access resources behind the firewall.
 
-On the server-side, Teleport is a single binary which enables convenient secure
-access to behind-NAT resources such as:
+You can read more about its architecture [here](https://goteleport.com/docs/architecture/).
 
-* [SSH nodes](https://goteleport.com/docs/getting-started/) - SSH works in browsers too!
-* [Kubernetes clusters](https://goteleport.com/docs/kubernetes-access/introduction/)
-* [PostgreSQL, MongoDB, CockroachDB and MySQL databases](https://goteleport.com/docs/database-access/introduction/)
-* [Internal Web apps](https://goteleport.com/docs/application-access/introduction/)
-* [Windows Hosts](https://goteleport.com/docs/desktop-access/introduction/)
-* [Networked servers](https://goteleport.com/docs/server-access/introduction/)
+We have implemented Teleport as a single Golang binary that integrates with multiple protocols and cloud services:
 
-Teleport is trivial to set up as a Linux daemon or in a Kubernetes pod. It's rapidly
-replacing legacy `sshd`-based setups at organizations who need:
+* [SSH nodes](https://goteleport.com/docs/getting-started/).
```

Comment:
Not sure why we're linking to this page for "SSH nodes" - it hardly mentions SSH.

Also, we've already included this link on line 19 so we probably don't need to include it twice in such a short document.